### PR TITLE
Modernize and updates from MK's local install

### PR DESCRIPTION
**Pull request by mkitt**
Thought I'd leak back what I have running in my local environment. The commit messages are fairly detailed and tell a pretty good story on the changes, especially if you are just maintaining your own version manually.

Let me know if you have any questions.

/cc @rynbyjn @jayzes

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -62,11 +62,6 @@ well put it to good use.
 - From the drop down, select `^ Control` under the `Caps Lock` setting
 - In the `Select Keyboard` drop down, set it for both internal and external keyboards
 
-### Fonts
-
-You look at text all damn day. Do yourself a favor and install the
-[MesloGM font][font] and rock the medium variant, size `12`.
-
```
Comment (jayzes) [inline Readme.md:69]: No more Meslo? What do you use these days?

Comment (mkitt) [inline Readme.md:69]: I added some notes in [this commit](https://github.com/ello/dotfiles/commit/bf89f7db0017f72f443f4941d152e4f7c4514c5f). Basically all Meslo did was increase the line-heights on Apple's Menlo font. You can now control this in your terminal which the polarized.terminal file has the settings for theme.

**Comment by mkitt** [comment]
I have a few more changes to make here. Might be a best to wait and merge until after the holiday break.

:christmas_tree:

**Comment by rynbyjn** [comment]
LGTM :metal:

**Pull request merged**